bigframes/dataframe.py

```diff
@@ -1062,12 +1062,31 @@ def drop(
             level_id = self._resolve_levels(level or 0)[0]
 
             if utils.is_list_like(index):
-                block, inverse_condition_id = block.apply_unary_op(
-                    level_id, ops.IsInOp(index, match_nulls=True)
-                )
-                block, condition_id = block.apply_unary_op(
-                    inverse_condition_id, ops.invert_op
-                )
+                # Only tuple is treated as multi-index value combinations
+                if isinstance(index, tuple):
+                    if level is not None:
+                        raise ValueError("Multi-index tuple can't specify level.")
+                    condition_id = None
+                    for i, idx in enumerate(index):
+                        level_id = self._resolve_levels(i)[0]
+                        block, condition_id_cur = block.apply_unary_op(
+                            level_id, ops.partial_right(ops.ne_op, idx)
+                        )
+                        if condition_id:
+                            block, condition_id = block.apply_binary_op(
+                                condition_id, condition_id_cur, ops.or_op
```

Comment:
Wondering if this should be `ops.and_op`. Does the example in https://b.corp.google.com/issues/317131784#comment1 gets fixed after this change? Would be nice to include that as an inline test case.

Reply:
It is or. We keep the rows based on the condition to be True, which is: none of the columns matches the drop value. The example is fixed (I pasted the link in the bug, since the we shouldn't paste link on github). It will be redundant to add the test case as the test suite covered.